docker/domains-analytics-db/domains-init.sql

```diff
@@ -549,6 +549,7 @@ CREATE TABLE IF NOT EXISTS domains.eservice_template_risk_analysis (
   created_at TIMESTAMP WITH TIME ZONE NOT NULL,
   risk_analysis_form_id VARCHAR(36) UNIQUE NOT NULL,
   risk_analysis_form_version VARCHAR(2048) NOT NULL,
+  tenant_kind VARCHAR NOT NULL,
```

Comment:
This table is under Redshift, and VARCHAR without length throws some issue during the data persistence. 

```suggestion
  tenant_kind VARCHAR(2048) NOT NULL,
```